apps/omg_eth/lib/omg_eth/root_chain.ex

```diff
@@ -141,7 +141,8 @@ defmodule OMG.Eth.RootChain do
   end
 
   @decorate measure_event()
-  @spec get_piggybacks(non_neg_integer, non_neg_integer, optional_addr_t) :: {:ok, [in_flight_exit_piggybacked_event]}
+  @spec get_piggybacks(non_neg_integer, non_neg_integer, optional_addr_t) ::
+          {:ok, [in_flight_exit_piggybacked_event]} | {:error, :failed_to_get_ethereum_events}
```

Comment:
The `dialyzer` had an issue with this spec, so I updated it to include the `:error` result.